### PR TITLE
Coverage visualisation fixes

### LINKED ISSUE #182: Improve coverage visualisation in the gutter
# Description
The client would like to make it clearer that clicking next to each line can yield different results, for example by making the gutter show boxes next to each line (e.g. by adding a border in the UI if that's possible) instead of a single continuous green line for consecutive covered lines.

# Definition of Done
* [x] The concerns above are addressed

### LINKED ISSUE #185: Open tool window when pressing on test from gutter
# Description
When a test case or mutant link is clicked on in the gutter, the tool window should open automatically and if possible the tool window should scroll to the relevant test case.

# Definition of Done
* [x] When clicking on the link to highlight a test case, open the tool window tab displaying generated test cases
* [x] When clicking on the link for a mutant, open the tool window displaying generated test cases
* [x] When highlighting a test case, scroll to it
* [x] When highlighting all test cases killing a mutant, scroll to an arbitrary test case that kills this mutant
* [ ] Cover edge cases, such as attempting to highlight a deleted test case, or no remaining test cases killing a mutant

### PR DESCRIPTION
**Pull request by kirilvasilev16**
# Description of changes made
ToolWindow is now being open when you select a test.
The scrollbar moves to the correct test cases when selecting some from the gutter.
The gutter has been changed to be more distinguishable than before.

# Why is merge request needed
The client requested those changes last meeting.

# Other notes
Closes #182
Closes #185 

- [x] I have checked that I am merging into correct branch

### CODE REVIEW

File: src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/TestCaseDisplayService.kt [hunk 1]
```diff
@@ -229,6 +233,36 @@ class TestCaseDisplayService(private val project: Project) {
...
+            toolWindowManager.contentManager.setSelectedContent(content!!)
+        }
+    }
+
+    /**
+     * Scrolls to the highlighted panel.
+     *
+     * @param myPanel the panel to scroll to
+     */
+    private fun scrollToPanel(myPanel: JPanel) {
```
Comment (jzelenjak) [inline src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/TestCaseDisplayService.kt:253]: Pretty cool approach

**Review by jzelenjak — APPROVED** [review]
The fixes and changes that you made work but sometimes there is some graphics artefacts that i am not sure are possible to fix. Other than that, good job!

**Review by bolkhod — APPROVED** [review]

File: src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/TestCaseDisplayService.kt (hunk 1 above)
Comment (kirilvasilev16) [reply src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/TestCaseDisplayService.kt:253]: :)
